src/components/DayPickerRangeController.jsx

```diff
@@ -401,11 +388,18 @@ export default class DayPickerRangeController extends React.Component {
           if (didFocusChange || recomputeDayBlocked) {
             if (isDayBlocked(momentObj)) {
               modifiers = this.addModifier(modifiers, momentObj, 'blocked-calendar');
+              isBlocked = true;
             } else {
               modifiers = this.deleteModifier(modifiers, momentObj, 'blocked-calendar');
             }
           }
 
+          if (isBlocked) {
```

Comment:
this is covering isDayBlocked and isOutsideRange, but `this.isBlocked` also covers `doesNotMeetMinimumNights`. should that get a test case also?

Reply:
That gets covered in a minimum nights specific code path because the triggers for re-calculating min nights are slightly different. It does get handled though!